gr-blocks/lib/annotator_raw_impl.h

```diff
@@ -13,15 +13,15 @@
 
 #include <gnuradio/blocks/annotator_raw.h>
 #include <gnuradio/thread/thread.h>
-
+#include <set>
 namespace gr {
 namespace blocks {
 
 class annotator_raw_impl : public annotator_raw
 {
 private:
     const size_t d_itemsize;
-    std::vector<tag_t> d_queued_tags;
+    std::set<tag_t> d_queued_tags;
```

Comment:
this needs a custom comparator, because `gr::tag_t::operator<(left, right)` only compares offsets. Or we just don't care about mutliple "comparator-wise identical" tags, and use a `std::multiset` instead.

```suggestion
    std::multi_set<tag_t> d_queued_tags;
```

I believe that changes nothing about the other functionality.